Backend/Tournament_Admin/TournamentAdminService/src/main/java/TournamentAdminService/controller/TournamentController.java

```diff
@@ -0,0 +1,96 @@
+package TournamentAdminService.controller;
+
+import TournamentAdminService.response.ApiResponse;
+import TournamentAdminService.model.Tournament;
+import TournamentAdminService.service.TournamentService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/admin/tournaments")
```

Comment:
might need to rename routes cos this violates route obsfucation kinda ( not revealing admin endpoints ) TBC

Reply:
done